src/migrateDiagram.js

```diff
@@ -320,15 +320,20 @@ function migrateDI(definitions, moddle) {
       }
 
       if (is(extensionElement, 'biodi:Edge')) {
+        const dmnElementRef = getDMNElementRef(semantic, extensionElement.get('source'));
+
+        // DMN element does exist

```

Comment:
Does or doesn't?

Reply:
Let's leave this typo in here and see how long it takes until someone notices.

Reply:
Lol.

Reply:
I've updated the PR as I'm not sure what _Lol._ refers to.